YvesIraguha/TeaHouse
# #170446825 Send Work to Admin

#### What does this PR do?
- Enable user to send the work to admin
#### Description of Task to be completed?
- Validate the file being sent
-  Send the email to an admin
#### How should this be manually tested?
- Clone the repo and checkout the branch `ft-submit-work-attachments-170446825`
- Run `npm i && npm run precommit && npm run dev:start` to run tests and start the server
- Consume the endpoint `/api/v1/users/submission`, and you should see the process for sending your work

#### What are the relevant pivotal tracker stories?
#[170446825](https://www.pivotaltracker.com/story/show/170446825)
